apps/showcase/src/components/showcase/component-replacement/component-replacement-pres.template.html

```diff
@@ -1,6 +1,6 @@
 <div class="card p-2 flex-row">
   <div class="w-50">
-    <p>Gregorian date: <b>{{dateFormControl.value}}</b></p>
+    <p>Gregorian date: <b class="e2e-ignore">{{dateFormControl.value}}</b></p>
```

Comment:
can we use some other class for the like `visual-testing-ignore`. I feel that `e2e-ignore` is a bit misleading  because we can still use these elements in tests.

Reply:
yes we can, I've used this classname because it's the one we use by default for our visual-testing helpers